cpp/tests/reshape/interleave_columns_tests.cpp

```diff
@@ -374,4 +368,298 @@ TYPED_TEST(FixedPointTestBothReps, FixedPointInterleave)
   }
 }
 
+namespace {
+using StrListsCol = cudf::test::lists_column_wrapper<cudf::string_view>;
+using IntListsCol = cudf::test::lists_column_wrapper<int32_t>;
+using IntCol      = cudf::test::fixed_width_column_wrapper<int32_t>;
+using TView       = cudf::table_view;
+
+constexpr bool print_all{true};  // For debugging
+constexpr int32_t null{0};
+
+auto null_at(cudf::size_type idx)
+{
+  return cudf::detail::make_counting_transform_iterator(0, [idx](auto i) { return i != idx; });
+}
+auto null_at(std::vector<cudf::size_type> const& indices)
```

Comment:
Same as above: `iterator_with_null_at()` is available in `cudf_test/iterator_utilities.hpp`.